src/System.Security.Cryptography.X509Certificates/tests/PfxTests.cs

```diff
@@ -104,6 +105,59 @@ public static void ExportWithPrivateKey()
             }
         }
 
+        [Fact]
+        public static void ReadECDsaPrivateKey_WindowsPfx()
+        {
+            using (var cert = new X509Certificate2(TestData.ECDsaP256_DigitalSignature_Pfx_Windows, "Test"))
+            using (ECDsa ecdsa = cert.GetECDsaPrivateKey())
+            {
+                Assert.NotNull(ecdsa);
+
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    AssertEccAlgorithm(ecdsa, "ECDSA_P256");
+                }
+            }
+        }
+
+        [Fact]
+        public static void ReadECDsaPrivateKey_OpenSslPfx()
+        {
+            using (var cert = new X509Certificate2(TestData.ECDsaP256_DigitalSignature_Pfx_OpenSsl, "Test"))
+            using (ECDsa ecdsa = cert.GetECDsaPrivateKey())
+            {
+                Assert.NotNull(ecdsa);
+
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Same question as above.  Seems like this is actually what you intended and I'm just missing some key piece of the puzzle.